build/root/.bazelrc

```diff
@@ -1,5 +1,8 @@
 startup --expand_configs_in_place
 
+# Force us to use python2
+build --host_force_python=PY2
```

Comment:
Which things need py2? Can we link to issues like we did for https://github.com/kubernetes/test-infra/blob/master/.bazelrc#L8-L10

Reply:
off the top of my head https://github.com/bazelbuild/rules_docker/issues/842#issuecomment-508263187 (hey look, you're on this thread! :upside_down_face: ), not sure if that's everything though.

Reply:
https://github.com/bazelbuild/rules_docker#setup would also be a good note ("NEW: Starting with Bazel 0.27.0, you also need to add to your .bazelrc file the following: ...")

Reply:
ref https://github.com/bazelbuild/bazel/issues/7899 https://github.com/bazelbuild/rules_docker/issues/842
/lgtm
/approve